### ISSUE TITLE
 TI 83 Premium CE doesn't leave exam mode #104 

### ISSUE THREAD

**Issue by PWDSerialgamer07** [adriweb/webcalclink#1]
Running on endeavourOS, already did what's mentioned in [this ](https://github.com/webusb/arduino/issues/29#issuecomment-299391705)issue, and tried to exit exam mode. Here is the raw frames log output:
```
22:48:12.491 > 54FD64004003640140031343040001FB6401FF00
22:48:12.494 > 54FD640080016401409031AB0A00015200000004010000000000
```

**Comment by debrouxl** [comment]
Well... These raw frames contain NavNet protocol packets, i.e. packets suitable for most calculators in the Nspire series (*). However, the 83PCE is a TI-eZ80 calculator, which talks the CARS ("DUSB") protocol, like all USB-capable TI-Z80 and TI-eZ80 calculators, as well as the USB-capable member of the TI-68k series, namely the 89T :)

*: in their default configuration; the CX II family defaults to NNSE, which wraps a variant of NavNet, but can be requested to talk NavNet as well.

**Comment by adriweb** [comment]
Yeah a packet/frame to send to the CE models in order to leave exam mode aren't implemented for now. That said, considering it's just a matter of sending any file to the calc, you can use any transfer program (including web-based ones like https://ticalc.link) to do so.
